### PR TITLE
fix: Allowed basic number field to start with a lone minus char

### PR DESCRIPTION
**Pull request by Nikeleos**

### CODE REVIEW

File: src/BasicInputs/components/NumberFormatCustom.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ function NumberFormatCustom (props) {
 
   const valuesRange = (inputObj) => {
     const { value } = inputObj;
-    if (min <= value && value <= max) return inputObj;
+    if (min <= value && value <= max || min < 0 && value === '-') return inputObj;
```
Comment (COSMO-VMI) [inline src/BasicInputs/components/NumberFormatCustom.js:13]: Could you please add parenthesis, so the operators priority is clear?
I assume you meant `(min <= value && value <= max) || (min < 0 && value === '-')`
Nontheless, `(min < 0 && value === '-')` doesn't seem right: indeed, '-' is not a number, and therefore can't be compared to a number.
I'd suggest the following:
`value === '-' || (min <= value && value <= max)`

**Review by COSMO-VMI — CHANGES_REQUESTED** [review]

File: src/BasicInputs/components/NumberFormatCustom.js (hunk 1 above)
Comment (Nikeleos) [reply src/BasicInputs/components/NumberFormatCustom.js:13]: I fixed the comparisons order and placed parenthesis correctly

**Review by jreynard-code — APPROVED** [review]
LGTM ;)

File: src/BasicInputs/components/NumberFormatCustom.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ function NumberFormatCustom (props) {
 
   const valuesRange = (inputObj) => {
     const { value } = inputObj;
-    if (min <= value && value <= max) return inputObj;
+    if ((value === '-' && min < 0) || (min <= value && value <= max)) return inputObj;
```
Comment (COSMO-VMI) [inline src/BasicInputs/components/NumberFormatCustom.js:13]: LGTM!

**Review by COSMO-VMI — APPROVED** [review]

File: src/BasicInputs/components/NumberFormatCustom.js (hunk 1 above)
Comment (COSMO-VMI) [reply src/BasicInputs/components/NumberFormatCustom.js]: Thanks!

**Pull request merged**